mobile/lib/widgets/shared_link/shared_link_item.dart

```diff
@@ -77,7 +77,10 @@ class SharedLinkItem extends ConsumerWidget {
         return;
       }
 
-      Clipboard.setData(ClipboardData(text: "${serverUrl}share/${sharedLink.key}")).then((_) {
+      final linkIdentifier = sharedLink.slug ?? sharedLink.key;
+      final pathPrefix = sharedLink.slug != null ? 's/' : 'share/';
```

Comment:
Having two different cases here feels brittle. Something along the lines of `if (slug) 's/' + slug else 'share/' + key` (pseudocode ofc) might be better.